src/components/ScholarshipDetailCard.jsx

```diff
@@ -208,6 +209,7 @@ export default function ScholarshipDetailCard({ scholarship }) {
       </Box>
       <Chip
         component={Link}
+        href={getMailToLink({ subject: 'Bug Report', body: deviceInfo })}
```

Comment:
This should maybe be a different subject/body as it's specific to the scholarship.

Reply:
Also might want to `bcc` the author of the scholarship.